src/main/java/tutorlink/grade/Grade.java

```diff
@@ -0,0 +1,34 @@
+package tutorlink.grade;
+
+import tutorlink.component.Component;
+import tutorlink.student.Student;
+
+public class Grade {
+    private double score;
+    private Component component;
+    private Student student;
+
+    public Grade(Component component, Student student, double score) {
+        this.component = component;
+        this.student = student;
+        this.score = score;
+    }
+
+    public String getGradeMatricNumber(Student student) {
+        return student.getMatricNumber();
+    }
+
+    public String getGradeComponentDescription(Student student) {
+        return student.toString();
+    }
```

Comment:
too specific, return Student and Component would be better
replace getGradeMatricNumber -> getStudent
replace getGradeComponentDescription -> getComponent
both or just simple getter